catalog-be/src/main/java/org/openecomp/sdc/be/tosca/CsarUtils.java

```diff
@@ -1021,8 +1021,9 @@ private boolean checkComponentBeforeOperation(Component component) {
         for (final ArtifactGroupTypeEnum artifactGroupTypeEnum : artifactsInfo.keySet()) {
             final String groupTypeFolder = path + WordUtils.capitalizeFully(artifactGroupTypeEnum.getType()) + PATH_DELIMITER;
             final Map<String, List<ArtifactDefinition>> artifactTypesMap = artifactsInfo.get(artifactGroupTypeEnum);
-            for (final String artifactType : artifactTypesMap.keySet()) {
-                final List<ArtifactDefinition> artifactDefinitionList = artifactTypesMap.get(artifactType);
+            for (final Map.Entry<String, List<ArtifactDefinition> entry : artifactTypesMap.entrySet()) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error, insert &#34;&gt;&#34; to complete ReferenceType1